lib/src/component_declaration/component_base.dart

```diff
@@ -91,7 +91,7 @@ typedef TProps BuilderOnlyUiFactory<TProps extends UiProps>();
 ///
 /// Extends [react.Component].
 ///
-/// Related: [UiStatefulComponent]
+/// For strongly-typed state, mix in [UiStatefulMixin] or extend from [UiStatefulComponent].
```

Comment:
Would it be fair to say that using `UiStatefulMixin` is considered best practice, instead of extending `UiStatefulComponent`? If so should we just deprecate `UiStatefulComponent`?